site/docs/guide/filter-queries.md

```diff
@@ -129,7 +129,7 @@ The syntactic sugar `:me` can be used to refer to your bot from within a query,
 
 ```ts
 bot.on("message:new_chat_members:is_bot"); // a bot joined the chat
-bot.on("message:left_chat_member:me"); // your bot left a chat (was removed)
+bot.on("message:left_chat_member:me"); // your bot left a chat (or was removed)
```

Comment:
This changes the meaning. Are you sure the updates are received when a bot leaves by itself? I think you only get those messages when someone removes your bot, but I'm not sure.